wheretofindme/settings/base.py

```diff
@@ -146,7 +146,10 @@ def boolish(val):
 # Database
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 
-DATABASES = {"default": dj_database_url.config(default="postgres:///wheretofindme")}
+if os.name == "nt":
```

Comment:
I'm assuming this change here has to do with Windows Postgres not handling Ident auth by default?

Reply:
Ident auth on Windows requires installing a special program to act as a Windows Ident server, as it's not supported by Windows by default, and this was just quicker and easier. (You can also apparently just set the authentication method for local connections to "trust" in the config file, but that's very not recommended unless you trust everyone using your local machine and network.) This PR was *supposed* to be about PNGs, but I probably should have or still should add information about this to the readme in case anyone else tries to set this up on Windows in the future, although I'm not completely confident I know the best way to do things? (I'm still not sure if $(uuidgen) is generating an actually secret value for the DJANGO_SECRET_KEY environment variable, or if it's being set to `$(uuidgen)` as a string literal, as I've never seen that work that way on Windows.)